kodilla-hibernate/src/test/java/com/kodilla/hibernate/manytomany/facade/SearcherFacadeTestSuite.java

```diff
@@ -0,0 +1,80 @@
+package com.kodilla.hibernate.manytomany.facade;
+
+import com.kodilla.hibernate.manytomany.Company;
+import com.kodilla.hibernate.manytomany.Employee;
+import com.kodilla.hibernate.manytomany.dao.CompanyDao;
+import com.kodilla.hibernate.manytomany.dao.EmployeeDao;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SearcherFacadeTestSuite {
+    @Autowired
+    private SearcherFacade searcherFacade;
+
+    @Autowired
+    private EmployeeDao employeeDao;
+
+    @Autowired
+    private CompanyDao companyDao;
+
+    @Test
+    public void testFindEmployeesWhoseLastNameContainsPhrase() {
+        //Given
+        employeeDao.save(new Employee("John", "Smarth"));
+        employeeDao.save(new Employee("Stephanie", "Clarckson"));
+        employeeDao.save(new Employee("Linda", "Kovalsky"));
+        employeeDao.save(new Employee("Martha", "Smith"));
+
+        //When
+        List<Employee> employeesWithGivenPhraseInLastName = searcherFacade.findEmployeesWhoseLastNameContainsPhrase("Sm");
+        List<String> fullNames = employeesWithGivenPhraseInLastName.stream()
+                .map(Employee::getLastName)
+                .collect(Collectors.toList());
+        List<Employee> employeesWithGivenPhraseInLastNameError = searcherFacade.findEmployeesWhoseLastNameContainsPhrase("Svi");
```

Comment:
1. Czemu to ma w nazwie `error`? Przecież to nie błąd, po prostu szukasz nazwy, której nie ma w bazie.
2. To się raczej nadaje na odrębny test, np. `shouldReturnEmptyListWhenNoEmployeesFound`. Podobnie w poniższym teście.